nodejs/week3/calculator/index.js

```diff
@@ -0,0 +1,42 @@
+const express = require("express");
+const app = express();
+
+
+const calculator = function (request, response) {
+    const numbersStringfied = Object.values(request.query);
+        console.log("numbersStringfied", numbersStringfied);
+    const merged = [].concat.apply([], numbersStringfied);
+    const numbersArray = merged.map(Number);
+        console.log("numbersArray", numbersArray);
+        console.log("params", request.params);
+
+    if (request.params.operation === 'add') {
+        const result = numbersArray.reduce((total, value) => total + value);
+            console.log("sum", result);
+        response.send(`The result os the SUM operation with the values ${numbersStringfied} is <b>${result}</b>`);
+    } else if (request.params.operation === 'subtract'){
+        const result = numbersArray.reduce((total, value) => total - value);
+            console.log("sub", result)
+        response.send(`The result os the SUB operation with the values ${numbersStringfied} is <b>${result}</b>`);
+    } else if (request.params.operation === 'multiply'){
+        const result = numbersArray.reduce((total, value) => total * value);
+            console.log("mult", result)
+        response.send(`The result os the MULT operation with the values ${numbersStringfied} is <b>${result}</b>`);
+    } else if (request.params.operation === 'divide') {
+        const result = numbersArray.reduce((total, value) => total / value);
+            console.log("div", result)
+        response.send(`The result os the DIV operation with the values ${numbersStringfied} is <b>${result}</b>`);
```

Comment:
You probably want to `send` only the result, and not the string with the explanation. You do the explanation on the other side. What if i want to use this number somewhere else? Then i would have to extract it from the string again.